.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
```

Comment:
The `no-proto` rule is set to `0`, which disables it. This might allow the use of deprecated `__proto__` properties. Consider enabling this rule to prevent potential issues with prototype pollution.